File: assignment 10 - C# practice problems/33.cs
//Write a c# program to add items in an blank array and display the items.

using System;
using System.Collections.Generic;

namespace MyNameSpace
{
    class Program33
    {
        public static void AddItemsToArrayDriverFunction()
        {

            var numArray = new List<int>() { };
            Console.WriteLine("array is " + "[{0}]", string.Join(",", numArray));
            Console.WriteLine("Give array eleents seperated by commas ( , )");

            string input = Console.ReadLine();

            var split = input.Split(",");
            foreach (var item in split)
            {
                numArray.Add(int.Parse(item));

            }

            Console.WriteLine("array is " + "[{0}]", string.Join(",", numArray));




        }


    }
}

File: assignment 3 - js practice problems/web-dev-starter/33.js
//Write a JavaScript program to add items in an blank array and display the items.
function addItemsToArray(arrayItemsParam) {
    arrayElement.push(arrayItemsParam)
    return arrayElement;
}

function addItemsToArrayDriverFunction() {
    let arrayElement = [];
    let arrayItemsToadd = prompt("array is: [ ]" + arrayElement + "\nEnter items to add in array" +
        "seperated by comma ( , )");

    alert("added item array " + addItemsToArray(arrayItemsToadd.split(",")));
}
addItemsToArrayDriverFunction();